server/channels/app/plugin_install.go

```diff
@@ -220,6 +221,15 @@ func (ch *Channels) installPlugin(bundle, signature io.ReadSeeker, installationS
 		logger.Warn("Failed to notify plugin status changed", mlog.Err(err))
 	}
 
+	ch.cfgSvc.UpdateConfig(func(cfg *model.Config) {
+		if _, ok := cfg.PluginSettings.Plugins[manifest.Id]; !ok {
+			cfg.PluginSettings.Plugins[manifest.Id] = make(map[string]any)
+
+			for _, pluginSetting := range manifest.SettingsSchema.Settings {
```

Comment:
This is the line that panics in the tests. Not every plugin has a `SettingsSchema` field. You need to check for that first.